Assets/Scripts/GlobalRunes.cs

```diff
@@ -46,16 +51,19 @@ void Update()
         }
     }
 
-    public static void Test()
+    public void Test()
     {
         Debug.Log("Test successful!");
     }
 
 
     // AddQueue adds a rune and the desired update to the queue
-    public static void AddQueue(GameObject rune, string mode, float rot, Vector2 point)
+    public void AddQueue(GameObject rune, string mode, float rot, Vector2 point)
     {
-        Debug.Log(mode);
+        if (updateQ == null)
```

Comment:
Why are you checking again if it's null?

Reply:
(Same answer as above) So for this one, I had a lot of issues with an "Object null reference" which I think was occurring because a Rune script was trying to access updateQ before the Start function of GlobalRunes was called. So to fix this, I made it check if it needed to create the queue before it needed to do something with it. There might be a way to make it work for Start, but this solution was simple and fixed the issue.